get-started/codelab/index.md

```diff
@@ -207,92 +206,83 @@ You can find the
 package, as well as many other open source packages, on
 [pub.dartlang.org](https://pub.dartlang.org/flutter/).
 
-<ol markdown="1">
-
-<li markdown="1"> The pubspec file manages the assets for a Flutter app.
+ 1. The pubspec file manages the assets for a Flutter app.
     In **pubspec.yaml**, add **english_words** (3.1.0 or higher)
     to the dependencies list.
     The new line is highlighted below:
 
-<!-- skip -->
-{% prettify dart %}
-dependencies:
-  flutter:
-    sdk: flutter
+    {% prettify yaml %}
```

Comment:
Let's talk about prettify offline. I was using prettify, as I always do, when I realized that few of the engineers have been using it (most docs use tick-tick-tick), and the TVC tech writer made a passionate argument for not using prettify, unless she needs code highlighting. I finally gave in.

Reply:
> using prettify, unless she needs code highlighting

I agree with that statement :). Markdown fenced code blocks are processed by [Rouge](https://github.com/jneen/rouge) at site compile time, which I think is nice.

I'm unsure what the objections were to prettify, but I've improved the plugin a bit (and can make further changes as necessary).